frontend/src/features/properties/components/forms/subforms/PaginatedFormErrors.tsx

```diff
@@ -3,19 +3,40 @@ import { useFormikContext } from 'formik';
 
 interface PaginatedFormErrorsProps {
   errors: number[];
-  nameSpace: string;
+  pagingRef?: any;
 }
 
 /**
  * Component that displays a csv list of errors for paginated forms if submit fails.
- * @param errors a list of pages that are in errors.
+ * @param errors a list of pages that are in error.
  */
 const PaginatedFormErrors: React.FC<PaginatedFormErrorsProps> = ({
   errors,
-  nameSpace,
+  pagingRef,
 }: PaginatedFormErrorsProps) => {
   const formik = useFormikContext();
   const [submitAttempted, setSubmitAttempted] = useState(false);
+  const [forceNavigation, setForceNavigation] = useState(true);
+
+  //This will ensure the dom has finished updating before manually updating the dom.
+  setTimeout(() => {
+    if (pagingRef?.current && submitAttempted) {
```

Comment:
always clear error classes to make sure that resolved errors are updated visually.